packages/rocketchat-api/server/routes.coffee

```diff
@@ -110,3 +110,65 @@ RocketChat.API.v1.addRoute 'groups.list', authRequired: true,
 	get: ->
 		roomIds = _.pluck RocketChat.models.Subscriptions.findByTypeAndUserId('p', @userId).fetch(), 'rid'
 		return { groups: RocketChat.models.Rooms.findByIds(roomIds).fetch() }
+
+# List all users
+RocketChat.API.v1.addRoute 'users.list', authRequired: true,
+	get: ->
+		if RocketChat.authz.hasRole(@userId, 'admin') is false
+						return RocketChat.API.v1.unauthorized()
```

Comment:
I believe there is a little too much space here compared to the rest of the file's indentation.